vision_agent/agent/vision_agent_v2.py

```diff
@@ -162,12 +201,15 @@ def run_plan(
     exec: Execute,
     code: str,
     tool_recommender: Sim,
+    long_term_memory: Optional[Sim] = None,
```

Comment:
How would one provide a long_term_memory ? dump the df to csv and retrieve it later ?

Reply:
Yeah basically, you can save a `Sim` object from a previous run and load it and use it on a later run. At some point we can provide pre-built `Sim` objects, or `long_term_memory`, for people to use.